locales/de/subtitles/erwin-schulz/erwin-schulz-02.yml

```diff
@@ -157,51 +154,49 @@ de:
           s: 30
           from: 00:02:43,29
           to: 00:02:47,08
-          text: I wanted to become a type setter.
+          text: Also, ich wollte da Schriftsetzer werden.
         sequence-31:
           s: 31
           from: 00:02:47,08
           to: 00:02:50,26
-          text: My uncle sought for a place – rejected.
+          text: Mein Onkel hat sich auch drum bemüht – abgelehnt.
         sequence-32:
           s: 32
           from: 00:02:50,26
           to: 00:02:57,02
-          text: 'Then another uncle said: “Well, then you become a bricklayer” – as
-            well, no success.'
+          text: Dann hat ein anderer Onkel gesagt: „Na, dann wirst du Maurer - auch nichts.
```

Comment:
same as in line 37, the non-quoted text with the colon breaks the YAML format.
And the closing quotes are missing.